### PR TITLE
Add some automated tests for Hotline website

### PR DESCRIPTION
**Pull request by ShkarupaDC**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import re
+import unittest
+from typing import NoReturn
```
Comment (byteofmydream) [inline test.py]: not sure if using this library will bring any benefit, as all test methods always return no value

File: test.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import re
+import unittest
```
Comment (byteofmydream) [inline test.py]: for e2e testing recommend using 'pytest' or 'behave' library

File: test.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import TimeoutException
+
+
+class HotlineTestCase(unittest.TestCase):
+    def setUp(self) -> NoReturn:
+        self.driver = webdriver.Firefox()
+
+    def get_with_wait(
```
Comment (byteofmydream) [inline test.py]: try to avoid having helper functions in the same class/file as tests

File: test.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    def test_filter_by_option(self, option: str = "1920-2000") -> NoReturn:
+        self.driver.get("https://hotline.ua/computer/diski-ssd/")
+
+        option_link = self.get_with_wait(
+            xpath=f'//a[contains(text(), "{option}")]'
+        )
+        option_link.click()
+
+        label = option_link.get_attribute("data-eventlabel")
+        match = re.match(r'^.*\[(\d+)\]$', label)
```
Comment (byteofmydream) [inline test.py]: matching expected values using regular expressions introduces additional complexity

File: test.py (hunk 1 above)
Comment (ShkarupaDC) [reply test.py]: It's commonly used technique to use type hints in such cases to improve readability

File: test.py (hunk 4 above)
Comment (ShkarupaDC) [reply test.py]: I use regex to extract option id from string like this "option[id]". Could you explain why I shouldn't approach this task in such way?

File: test.py (hunk 1 above)
Comment (byteofmydream) [reply test.py]: it doesn't improve anything as test functions never return any value

File: test.py (hunk 4 above)
Comment (byteofmydream) [reply test.py]: even if you really want to do that you should extract this action to a separate function. but still, I don't see a reason for that as e2e tests should interact with SUT in the same way as a user does, avoiding any kind of low-level interactions with DOM except when there is no other way to perform a required action

File: test.py (hunk 2 above)
Comment (ShkarupaDC) [reply test.py]: Should I switch to one of these libraries?

File: test.py (hunk 4 above)
Comment (ShkarupaDC) [reply test.py]: Thank you. In this situation, I should check if an option is added to the selected list after a click on it. I can do that by matching strings stored in the 'innerText' attribute or using option id. What is the appropriate way to do that?

Comment (byteofmydream) [reply test.py]: you should use this approach only if you can't do this with webdriver .text attribute

**Pull request merged**